low-level/sensors/impl/centos_7/service_monitor.py

```diff
@@ -114,6 +127,30 @@ def initialize(self, conf_reader, msgQlist, product):
         self._manager = Interface(systemd,
                             dbus_interface='org.freedesktop.systemd1.Manager')
 
+        cache_dir_path = os.path.join(DATA_PATH, self.CACHE_DIR_NAME)
+        self.SERVICE_MONITOR_DATA_PATH = os.path.join(cache_dir_path,
+                                         f'SERVICE_MONITOR_DATA')
```

Comment:
should I add node_id at the end of the cache file name?

Reply:
If we have added node_id for other files, then I think we should add here as well.

Reply:
done.